src/lib/structures/schemas/defaultGuildSchema.js

```diff
@@ -64,23 +49,62 @@ module.exports = KlasaClient.defaultGuildSchema
             .add("PROFANITY", "any", { default: { enabled: false, threshold: 0.99 } }))
         .add("invites", "boolean", { default: false }))
 
-    // Self Roles
-    .add("selfroles", folder => folder
-        .add("roles", "role", { array: true })
-        .add("enabled", "boolean", { default: true }))
-
-    // Level Roles
-    .add("levelroles", folder => folder
-        .add("roles", "any", { array: true, configurable: false })
-        .add("enabled", "boolean", { default: true }))
-
     // Patreon
     .add("patreon", folder => folder
         .add("enabled", "boolean", { default: false, configurable: false })
         .add("patron", "user", { configurable: false }))
 
+// New Schema Start -------
+
     // Misc
-    .add("musicVolume", "integer", { default: 100, max: 100, min: 0 })
-    .add("djOnly", "boolean", { default: false })
-    .add("levelup", "boolean", { default: false })
-    .add("leveltype", "string", { default: "guild", configurable: false });
+    .add("misc", folder => folder
+        .add("leveluptype", "string", { default: "guild", configurable: false })
+        .add("volume", "integer", { default: 100, max: 100, min: 0 }))
+
+    // Mod Logs
+    .add("modlogs", folder => folder
+        .add("logs", "any", { array: true, configurable: false })
+        .add("toggles", logs => logs
```

Comment:
Why would you disable modlog entries.

Anything related to modlogs should be logged. server logs are different and should have toggles.